docs/adr/0003-tinyvex-local-sync-engine.md

```
@@ -1,7 +1,7 @@
 # ADR 0003 — Tinyvex as the Local Sync Engine (SQLite + WS)
 
  - Date: 2025-10-31
- - Status: Accepted
+ - Status: Accepted — Updated for two‑package client and bridge‑hosted writer
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 1

(MD007, ul-indent)